packages/fast-form-generator-react/src/utilities/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./form-input.colors";
+export * from "./form-input.style";

```

Comment:
If you make the above change for the colors file I think you can remove this file and the utilities folder. Ideally form should contain all the necessary items for its rendering

Reply:
By moving these into the form folder its causing an issue that @nicholasrice and I investigated for awhile. I think we should save this for another PR.